lib/MetaProcessor/MetaParser.cpp

```diff
@@ -474,6 +474,34 @@ namespace cling {
     return false;
   }
 
+  // dumps/creates a trace of the requested representation.
+  bool MetaParser::istraceCommand() {
+    if (getCurTok().is(tok::ident) &&
+        getCurTok().getIdent().equals("trace")) {
+      consumeToken();
+      skipWhitespace();
+      if (!getCurTok().is(tok::ident))
+          return false;
+      std::string ident = getCurTok().getIdent();
+      consumeToken();
+      skipWhitespace();
+      if (getCurTok().is(tok::eof)) {
+        m_Actions->actOnstatsCommand(ident=="ast" 
+            ? llvm::StringRef("asttree") : llvm::StringRef(ident), llvm::StringRef());
```

Comment:
but here, since I didn't create an action for my "trace" cmd, I reuse the "stats" cmd, which requires asttree as second identifier...  so I have to remap........ I didn't touch the stats cmd and we wanted to avoid duplication
the syntax for trace is
.trace ast <filterstr> and for stats is .stats asttree ....

VK